sql/setup.sql

```diff
@@ -1,2 +1,23 @@
 -- Use this file to define your SQL tables
 -- The SQL in this file will be executed when you run `npm run setup-db`
+
+
+DROP TABLE IF EXISTS todos;
+DROP TABLE IF EXISTS userSL;
+
+CREATE TABLE userSL (
+    id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    email VARCHAR NOT NULL UNIQUE,
+    password_hash VARCHAR NOT NULL
+);
```

Comment:
Don't be afraid to name this table users as the DROP TABLE IF EXISTS users will allow the previous users table to be updated to what is currently in this sql file.